pipeline/emailing.py

```diff
@@ -10,34 +10,52 @@
 from load import create_connection
 
 
-def send_email(plant_id: int, value: float, value_type: str) -> None:
-    """Sends an email to the correct botanist if the plants conditions are worrying."""
+def send_email(plant_id: int, value: float, value_type: str, condition:str ) -> None:
+    """
+    Sends an email to the correct botanist if the plants conditions are worrying.
+    Example inputs - plant_id: 1, value: 10000, value_type: 'temperature', condition: 'exceeded'.
+    """
+
     load_dotenv()
-    html = generate_html(plant_id, value, value_type)
+    html = generate_html(plant_id, value, value_type, condition)
     client = boto3.client("ses", region_name="eu-west-2")
     message = MIMEMultipart()
-    message["Subject"] = "Plant ALERT!"
+    message["Subject"] = f"PLANT ALERT! ACCEPTABLE {value_type.upper()} LEVEL {condition.upper()}"
     body = MIMEText(
         html,
         "html")
     message.attach(body)
 
+    # client.send_raw_email(
+    #     Source=ENV['FROM_EMAIL'],
+    #     Destinations=[
+    #         ENV['TO_EMAIL1'],
+    #         ENV['TO_EMAIL2'],
+    #         ENV['TO_EMAIL3'],
+    #         ENV['TO_EMAIL4']
+    #     ],
+    #     RawMessage={
+    #         'Data': message.as_string()
+    #     }
+    # )
+
     client.send_raw_email(
-        Source=ENV['FROM_EMAIL'],
+        Source='trainee.andrew.mcwilliam@sigmalabs.co.uk',
```

Comment:
Could change this back to get it from the ENV when merging to main

Reply:
just before we submit this, ill change it to the botanist email, so we can send it directly to them